tests/unittests.c

```diff
@@ -67,11 +67,42 @@ static void cpu_freq_expect(const char *cpu_str, uint64_t expectation) {
     return;
 }
 
-static unsigned long test_kernel_task_func(void *arg) {
+static unsigned long test_kernel_task_func1(void *arg) {
     printk("CPU[%u]: Executing %s\n", smp_processor_id(), __func__);
     return _ul(arg);
 }
 
+static inline uint64_t rand47(void) {
+    return (_ul(rand()) << 16) ^ rand();
+}
+
+#define KT2_ROUNDS 100
+#define KT2_CHUNKS 10
+static unsigned long test_kernel_task_func2(void *arg) {
```

Comment:
nit: maybe rename the function to something like `test_kernel_pmm_vmm_stress_test`

Reply:
I tried to follow your naming schema for the tests cases, but I would also have preferred more describing names 🤷 Fixed in 6b1ebdc9cfacef472b23a47e141db6d56a4100d3